src/_pytest/faulthandler.py

```diff
@@ -9,6 +9,7 @@
 from _pytest.stash import StashKey
 
 
+fault_handler_original_stderr_fd_key = StashKey[int]()
```

Comment:
I'm starting to get the impression we should use attributes on a plugin object

Reply:
> I'm starting to get the impression we should use attributes on a plugin object

Could point to an example I can follow to implement? Thanks.

Reply:
It's More of a follow-up idea as we stash More and more items into the stash